cdap-adapters/src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -1,14 +1,5 @@
 CDAP-Adapter-Type: stream-conversion
```

Comment:
Do we still need these properties in the Manifest? Will they eventually be removed?